packages/resolvers/glob/src/GlobResolver.js

```diff
@@ -20,7 +20,10 @@ export default (new Resolver({
     let error;
     if (sourceAssetType !== 'js' && sourceAssetType !== 'css') {
       error = `Glob imports are not supported in ${sourceAssetType} files.`;
-    } else if (dependency.isURL) {
+    } else if (
+      dependency.specifierType === 'url' &&
+      !dependency.meta?.isCSSImport
```

Comment:
This came up when I rebased. We need some way to distinguish CSS `@import` from other URL dependencies in CSS files like `background: url(...)`. One supports globs and the other doesn't. Best I came up with for now was a meta property. Thoughts?